### PR TITLE
Oscillator+delay nodes

### PR DESCRIPTION
**Pull request by hughrawlinson**
Ok, so this contains a lot of changes, mostly by @ouhouhsami. This should not be merged right away, but I'm opening a pull request so we can discuss the problems with it and figure out how to fix them. From my testing, I can tell that OscillatorNode works (there are a few little discrepancies in behaviour between it and browser implementations, but they're not a major issue). DelayNode has a major problem to do with not having enough/the right channels. I'm working on that error now, hopefully I should have fixed it today.

### CODE REVIEW

File: npm-debug.log [hunk 1]
```diff
@@ -0,0 +1,1335 @@
+0 info it worked if it ends with ok
```
Comment (notthetup) [inline npm-debug.log]: You might want to remove this log file. :)

Comment (hughrawlinson) [inline npm-debug.log]: Done, oops.

**Comment by nickdbush** [comment]
It would be great to see this added!

**Comment by sebpiq** [comment]
@hughrawlinson I assume there hasn't really been any progress on this?

**Comment by hughrawlinson** [comment]
@sebpiq nope, sorry :/ I'll have another look when I have time.

**Comment by swordsreversed** [comment]
@hughrawlinson did you end up using this anywhere else? Really want to use that delay

**Comment by hughrawlinson** [comment]
@swordsreversed no, I didn't, sorry!

**Comment by swordsreversed** [comment]
@hughrawlinson no problem, i found this library https://github.com/mohayonao/web-audio-engine which has delayNode implemented

**Pull request closed without merging**